### PR TITLE
add responsive video

### PR DESCRIPTION
**Pull request by Arhell**
fix issue https://github.com/istio/istio.io/issues/10271

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [x] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: src/sass/base/_base.scss [hunk 1]
```diff
@@ -553,3 +553,7 @@ sup {
     margin-right: auto;
     max-width: 750px;
 }
+
+iframe {
+    max-width: 100%;
+}
```
Comment (rvennam) [inline src/sass/base/_base.scss:559]: needs a newline

Comment (Arhell) [reply src/sass/base/_base.scss:559]: Thanks, I will correct

**Comment by craigbox** [comment]
LGTM pending lint!

File: src/sass/base/_base.scss (hunk 1 above)
Comment (rvennam) [reply src/sass/base/_base.scss:559]:
```suggestion
}

```

**Comment by Arhell** [comment]
cc @craigbox

**Comment by craigbox** [comment]
Closes #10271.

**Review by craigbox — APPROVED** [review]

**Pull request merged**